nexong/api/Event/eventSerializer.py

```diff
@@ -118,7 +118,10 @@ def validate(self, attrs):
         validation_error = {}
 
         attendees_emails = attrs.get("attendees")
-        num_attendees = len(attendees_emails)
+        if attendees_emails is not None:
+            num_attendees = len(attendees_emails)
+        else:
+            num_attendees = 0
```

Comment:
I am a bit confused about this. If a list is empty, len(list) is equal to 0. You don't need to make an if statement for it. I mean, if you do `num_attendees = len(atendees_emails)`, num_attendees will be the size of atendees_emails, including 0. Did you do this because of an error?